### ISSUE TITLE
[ dress code ] A little bit about my terminal

### ISSUE THREAD

**Issue by rogeruiz** [rogeruiz/rogeruiz.github.io#11]
from @sharadgana [on Twitter](https://twitter.com/sharadgana/status/877416185733668865)
> Interested in your terminal setup. Can you share the details ? Thanks

In the interest of keeping things transparent. I'm gonna write up how I set up my terminal. I may even write an emotional essay about how important it is to me to have the tools I work with customized to just me and how important I think it is to not only customize them but also make it be the first thing you do even if it feels like a waste of time. (_spoiler alert it isn't_). When I do write that up, I'll link it to this work but will probably track it in a separate issue.

tl;dr for the technical bits.

- [x] talk about custom fonts 🔠 
- [x] talk about leveraging existing frameworks ( `oh-my-zsh` ) and themes ( `pure` ) 🏟 
- [x] talk about taking something you like and making it your own 🎨 
- [x] screenshot the hell out of it 📸
